git_machete/short_docs.py

```diff
@@ -0,0 +1,31 @@
+from typing import Dict
+
+short_docs: Dict[str, str] = {
```

Comment:
Okay... so in the target state of this repo, we'll have both short_docs.py and docs.py, the latter with**out** `short_docs` dict? in fact I'm for just having single docs.py, and removing short_docs.py completely. Single file is easier to manage

Reply:
Or actually... are these short docs generated out of .rst? or to be filled up manually

Reply:
So i created separate file short_docs to be modified manually and then used in the docs.py for the review process but ultimately i would not copy short_docs.py into docs.py as it is unnecessary, @PawelLipski ok?

Reply:
For clarification i can even rename the files to: short_docs_manually.py and docs_generated.py or sth like that, @PawelLipski wdyt?

Reply:
+1, I'd make it as clear as possible what's generated and what's manual

Reply:
Plus a comment at the beginning of each file to say whether it should or should not be modified